Description: Fix SDDM for uses for zsh shell
 code copied from kdm broke use with zsh
Author: Kevin Funk
Origin: upstream, https://github.com/sddm/sddm/pull/287
Bug: https://bugs.launchpad.net/ubuntu/+source/kdebase/+bug/13965
Forwarded: no, it's from upstream
Reviewed-by: Kevin Funk
Last-Update: 2014-10-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: sddm-0.9.0/data/scripts/Xsession
===================================================================
--- sddm-0.9.0.orig/data/scripts/Xsession
+++ sddm-0.9.0/data/scripts/Xsession
@@ -22,7 +22,7 @@ case $SHELL in
     ;;
   */zsh)
     [ -z "$ZSH_NAME" ] && exec $SHELL $0 "$@"
-    emulate -R zsh
+    emulate -R sh
     [ -d /etc/zsh ] && zdir=/etc/zsh || zdir=/etc
     zhome=${ZDOTDIR:-$HOME}
     # zshenv is always sourced automatically.
